src/components/MyKiva/MyKivaEmailUpdatesCard.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+	<div class="card-container tw-h-full">
+		<span
+			class="tw-inline-flex tw-items-center tw-gap-1 tw-mb-2
+						tw-rounded-md tw-bg-eco-green-1 tw-px-1.5 tw-py-0.5
+						tw-absolute tw-top-3 tw-left-3 tw-z-1"
+		>
+			<KvMaterialIcon
+				class="tw-w-2 tw-h-2 tw-shrink-0"
+				:icon="mdiEmailOutline"
+			/>
+			<span class="tw-text-primary tw-font-medium tw-align-middle" style="font-size: 0.875rem;">
+				Email updates
+			</span>
+		</span>
+		<KvBorrowerImage
+			class="tw-relative tw-w-full tw-overflow tw-rounded"
+			:alt="name"
+			:hash="hash"
+			:aspect-ratio="3 / 4"
+			:default-image="{ width: 304 }"
+			:images="[{ width: 304 }]"
+			:photo-path="$appConfig.photoPath"
+		/>
+		<h3 class="tw-pt-1 tw-text-nowrap">
+			Follow {{ borrowerName }}'s journey
+		</h3>
+		<p class="tw-font-medium tw-text-base tw-pt-0.5">
+			Don't miss out on borrower updates, stories, and new ways to help.
+		</p>
+		<div class="tw-pt-1">
+			<KvButton
+				variant="secondary"
+				v-kv-track-event="['portfolio', 'click', 'next-step-email-option']"
+				class="tw-w-full"
+				@click="handleEmailOptIn"
+			>
+				Get email updates
+			</KvButton>
```

Comment:
@dyersituations @tiara How does one "skip" or dismiss this step?

Reply:
Not a way currently as defined, but it's a good question. @tiarascoleman could you check in with Lauren if that's been discussed? Thanks!

Reply:
I see in the design there is a confirmation step if you click to opt-in and it has a link to Email preferences. So that's a good thing!

I suppose if this section isn't content that "cycles" and just sits here it's ok not to have a skip or dismiss option but it comes off as self serving on Kiva's part to take up so much space and not allow folks to dismiss it...